modules/perception/onboard/component/trafficlights_perception_component.cc

```diff
@@ -933,13 +926,12 @@ bool TrafficLightsPerceptionComponent::TransformDebugMessage(
 
   if (lights.size() > 0) {
     camera::CarPose pose;
-    if (GetCarPose(frame->timestamp, &pose)) {
-      Eigen::Matrix4d cam_pose;
-      pose.GetCameraPose("front_6mm", &cam_pose);
-      light_debug->set_distance_to_stop_line(stopline_distance(cam_pose));
-    } else {
-      AERROR << "error occured in calc distance to stop line";
+    if (!GetCarPose(frame->timestamp, &pose)) {
+      AERROR << "Error occured in calculate distance to stop line.";
     }
+    Eigen::Matrix4d cam_pose;
+    pose.GetCameraPose("front_6mm", &cam_pose);
+    light_debug->set_distance_to_stop_line(stopline_distance(cam_pose));
```

Comment:
Changed logic?

Reply:
Yep, but that's wrong. Also, maybe we don't need to show boolean text via boost, which is heavy but using '<< ( a ? "true" : "false")' style.

Reply:
Thanks! Then I will loop in the perception folks to review :)